### ISSUE TITLE
Trackers AvistaZ family have wrong pubDate

### ISSUE THREAD

**Issue by ghost** [Prowlarr/Prowlarr#1917]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

AvistaZ, CinemaZ and PrivateHD pubDates are all off by one hour into the past because the servers they're ran on seem to be out of daylight savings time now.

CinemaZ and PrivateHD are hosted on a server that experiences UTC-5 at the moment.

ExoticaZ pubDates are five hours into the future. Currently the code assumes the site is hosted in UTC-4, while they are currently supposed to be UTC+1 along with AvistaZ.

### Expected Behavior

The pubDates should be correct, even when DST starts/ends.

### Steps To Reproduce

1. Open RSS Feed
2. Notice pubDates are wrong

### Environment

```markdown
- OS: Debian 10
- Prowlarr: 1.9.4.4039
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: /
```

### What branch are you running?

Master

### Trace Logs?

<details>
<summary>Trace Logs</summary>
2023-11-11 22:29:14.2|Trace|Http|Req: 2074 [GET] /prowlarr/3/api?apikey=(removed)&extended=1&t=search (from 46.*.*.230 Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36)
2023-11-11 22:29:14.2|Debug|Prowlarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2023-11-11 22:29:14.3|Info|ReleaseSearchService|Searching indexer(s): [AvistaZ] for Term: [], Offset: 0, Limit: 0, Categories: []
2023-11-11 22:29:14.3|Debug|AvistaZ|Downloading Feed https://avistaz.to/api/v1/jackett/torrents?in=1&type=0&limit=50&search=
2023-11-11 22:29:14.3|Trace|IndexerHttpClient|Req: [GET] https://avistaz.to/api/v1/jackett/torrents?in=1&type=0&limit=50&search=
2023-11-11 22:29:14.3|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2023-11-11 22:29:14.4|Trace|IndexerHttpClient|Res: HTTP/2.0 [GET] https://avistaz.to/api/v1/jackett/torrents?in=1&type=0&limit=50&search=: 200.OK (53378 bytes) (100 ms)
2023-11-11 22:29:14.4|Trace|EventAggregator|Publishing IndexerQueryEvent
2023-11-11 22:29:14.4|Trace|EventAggregator|IndexerQueryEvent -> HistoryService
2023-11-11 22:29:14.4|Trace|EventAggregator|IndexerQueryEvent <- HistoryService
2023-11-11 22:29:14.4|Debug|ReleaseSearchService|Total of 50 reports were found for Term: [], Offset: 0, Limit: 0, Categories: [] from 1 indexer(s)
2023-11-11 22:29:14.4|Trace|Http|Res: 2074 [GET] /prowlarr/3/api?apikey=(removed)&extended=1&t=search: 200.OK (149 ms)
</details>

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by sgrochowski** [comment]
AnimeTorrents is in the AvistaZ family, although not following the namesake. The fix in da1965b resolved it for all my Z sites except for AnimeTorrents.

**Comment by sgrochowski** [comment]
> AnimeTorrents doesn't provide the hour on the browse page, so there's nothing to fix.

Ah, gotcha. Didn't know that's how scraping for AT worked, so thanks for that. Unfortunate as that affects some of my abilities to automate but AT's usability is so poor that it makes sense. 

Im seeing the same issue as the other 'Z' sites on DigitalCore that _does_ show hour on the browse page, but I can create a new issue for that since it's not a Z site.
